src/Components/Web.JS/src/Platform/BootConfig.ts

```diff
@@ -28,6 +29,9 @@ export interface BootJsonData {
   readonly cacheBootResources: boolean;
   readonly config: string[];
   readonly icuDataMode: ICUDataMode;
+
+  // These properties are tacked on, and not found in the boot.json file
+  modifiableAssemblies: string | null;
```

Comment:
```suggestion
  modifiableAssemblies:'debug' | null;
```

I think this could probably be an enum?

Reply:
Nice, I did not know that's how enums worked in TypeScript.

Reply:
Hmm, so the problematic part is that I know have to translate the response header

> Type 'string | null' is not assignable to type '"debug" | null'.

Since we don't super care about the header value, leaving this as-is is preferable.

Reply:
Ah bummber.